bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/F6_10/F6_10_00_EltakoFPE.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal.eep.F6_10;
+
+import static org.openhab.binding.enocean.internal.EnOceanBindingConstants.*;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.library.types.OpenClosedType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.enocean.internal.config.EnOceanChannelContactConfig;
+import org.openhab.binding.enocean.internal.eep.Base._RPSMessage;
+import org.openhab.binding.enocean.internal.messages.ERP1Message;
+
+/**
+ *
+ * @author Holger Englert - Initial contribution
+ */
+public class F6_10_00_EltakoFPE extends _RPSMessage {
+
+    final byte OPEN = 0x00;
+    final byte CLOSED = 0x10;
+
+    public F6_10_00_EltakoFPE() {
+        super();
+    }
+
+    public F6_10_00_EltakoFPE(ERP1Message packet) {
+        super(packet);
+    }
+
+    @Override
+    protected State convertToStateImpl(String channelId, String channelTypeId, State currentState, Configuration config) {
```

Comment:
```suggestion
    protected State convertToStateImpl(String channelId, String channelTypeId, Function<String, State> getCurrentStateFunc, Configuration config) {
```

Reply:
That should fix the compile error.

Reply:
Then I get the compile error `[...]F6_10_00_EltakoFPE.java:[43,1397] Function cannot be resolved to a type`

Reply:
I think you need to add some imports, too. Eclipse does that automatically on save. Did you rebase on master befor changing that? Otherwise you'll probably get another error.